packages/office-ui-fabric-react/src/components/Rating/Rating.tsx

```diff
@@ -70,24 +70,38 @@ export class Rating extends BaseComponent<IRatingProps, IRatingState> {
   }
 
   public render() {
+    let id = this._id;
     let stars = [];
-
-    for (let i = this.props.min as number; i <= (this.props.max as number); i++) {
+    let starIds = [];
+    let {
+      disabled,
+      getAriaLabel,
+      max,
+      min,
+      rating,
+      readOnly,
+      size
+    } = this.props;
+
+    for (let i = min as number; i <= (max as number); i++) {
       let ratingStarProps: IRatingStarProps = {
         fillPercentage: this._getFillingPercentage(i),
-        disabled: this.props.disabled ? true : false
+        disabled: disabled ? true : false
       };
 
+      starIds.push(this._getStarId(i - 1));
```

Comment:
not clear why you need an array to save the results, when you can just call the function in the one place you need the id...